multimodal/src/autogluon/multimodal/utils/from_labelstudio.py

```diff
@@ -0,0 +1,368 @@
+import json
+import os
+from enum import Enum
+
+import pandas as pd
+
+
+class TaskType(Enum):
```

Comment:
Can we reuse the constants as in https://github.com/autogluon/autogluon/blob/cc82a2e3e9b5d2578873d3501193b2794cd35273/multimodal/src/autogluon/multimodal/problem_types.py#L33

Reply:
There might be some inconsistency between this and the label-studio supported templates types (the overall supported types are in the links in https://labelstud.io/playground). I think it can be reused if we only want to support the problem types in the `constants`.

Reply:
Sounds good. Resolving this.